### DISCUSSION TITLE
Description for AI Models

### DISCUSSION THREAD

**Discussion by igeeksng** [open-webui/open-webui#4860] [Ideas]
Is there a possibility of adding short descriptions for downloaded AI models via the WebUI? Having concise explanations would greatly assist in understanding each model’s intended purpose when switching between them.

**Comment by silentoplayz** [comment]
This can already be done within the `Workspace` -> `Models` -> `Edit` option. Modelfiles can be provided with a `Description`.

**Reply by igeeksng** [reply]
This is awesome. Thank you
